### DISCUSSION TITLE
Problem in providing the proof for Student Developer Pack

### DISCUSSION THREAD

**Discussion by AgentXZ** [community/community#67293] [GitHub Education]
### Hi everyone,

Hello, I am contacting you because I have a problem with the student github program.
I tried several times to subscribe to this offer with my official documents which are schooling certificates, by renaming my account with my last name/first name and a short biography. 
Despite this, I still have a categorical refusal for the following reason: Unfortunately, we weren't able to approve your educational discount request for **** 
What happened?
Common verification problems and suggested solutions are documented here:
https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus- for-students/why-wasn’t-my-application-to-global-campus-for-students-approved.
I don't know at all how I could resolve this in order to be able to subscribe to this offer and be able to use its advantages for my courses.
I would like to point out that I had the correct first/last name before I just renamed myself.
Could you please help me ?
Thanks in advance,
Sincerely

**Comment by Wenrh2004** [comment]
Yes, I tried to promote the use of GitHub in my school and my personal software development team, but my academic credentials are buggy, including the main branch protection is not working, and most importantly I don't know how I should re-validate my academic affiliation, and, according to his instructions, I should ask GitHub support, but I've not been able to get any valid response, so I'm now have to consider whether to continue to rely on and promote this platform.

**Comment by AgentXZ** [comment]
I too can't figure out how to get out of this situation.
